.env.example

```
@@ -0,0 +1,20 @@
+PORT=3000
+REQUEST_TIMEOUT=30000
+
+# Database Configuration
+# Supported providers: sqlite (default), postgresql, mysql
+DATABASE_PROVIDER=sqlite
+
+# Database Connection URL
+# SQLite: file:../database/analytics.db
+# PostgreSQL: postgresql://user:password@localhost:5432/analytics?schema=public
+# MySQL: mysql://user:password@localhost:3306/analytics
+DATABASE_URL="file:../database/analytics.db"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)